### ISSUE TITLE
Assess fails if during collect an external file source is referenced as FileSystem needs to be remounted

### ISSUE THREAD

**Issue by paulbarr** [IBM/transformation-advisor-sdk#70]
**Describe the bug**
Assess stage fails

**To Reproduce**
In your plugin class override the getCollection() method and use a class you have created that implements the DataCollection interface. In this assign a value to the assessmentDataJsonFile that reference s a file resource OUTSIDE of the jar file
`this.assessDataJsonFile = Paths.get("/Users/paulbarry/values.json");`
The assess stage now fails because it can't mount the filesystem for files inside the jar anymore, and it needs it to read the complexity.json files etc

**Expected behavior**
Assess stage should work

**Screenshots**

**Environment (please complete the following information):**
SDK 0.5.4 and master

**Additional context**
I fixed this by changing the getRecommendation() method so that it remounts the FileSystem inside in the jar file

`String middlewareDir = "/" + this.getMiddleware() + "/";`
`final Map<String, String> env = new HashMap<>();    `
`final String[] array = getClass().getResource(middlewareDir + "complexity.json").toURI().toString().split("!");    `
`            //Because we read an input file from outside the jar we need to reset the filesystem to be able to read files inside the jar now.   `
`            final FileSystem fs = FileSystems.newFileSystem(URI.create(array[0]), env);    `
`           Path complexityJsonFile = fs.getPath((middlewareDir + "complexity.json"));    `
`            Path issueCatJsonFile = fs.getPath((middlewareDir + "issue-category.json"));    `
`            Path issueJsonFile = fs.getPath((middlewareDir + "issue.json"));    `
`            Path targetJsonFile = fs.getPath((middlewareDir + "target.json"));`

**Comment by transformationAdvisor-sdk** [comment]
@paulbarr ,  I think I have already fixed this problem on SDK 0.6.0.
In sample code, I can use this without any problem.
Will you check the SDK 0.6.0, to see your problem still there?

```
        Path assessDataJsonFile = new File("/Users/jianfeng@ca.ibm.com/sampleData/Installation1/"+assessmentUnitName+"/queueManager.json").toPath();
        List<Path> assessmentConfigFiles = new ArrayList<Path>();
        Path assessConfigJsonFile = new File("/Users/jianfeng@ca.ibm.com/sampleData/Installation1/"+assessmentUnitName+"/config.mqsc").toPath();
        assessmentConfigFiles.add(assessConfigJsonFile);
```

**Comment by transformationAdvisor-sdk** [comment]
This one is fixed.
